.github/workflows/lint.yml

```diff
@@ -42,3 +42,15 @@ jobs:
         if: success() || steps.check-format.conclusion == 'failure'
         run: |
           verible-verilog-lint --rules_config .github/workflows/config/verible.rules `cat files.txt`
+
+  lint_tcl:
+    name: Lint TCL scripts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v3
+
+      - name: Lint with tclint
+        run: |
+          python -m pip install tclint==0.1.0
```

Comment:
can you add this to the testing requirements for sc in the requirements.txt?

Reply:
done, I also tweaked the workflow to use `-c requirements.txt` to install just tclint based on the version in requirements.txt and avoid duplication (without requiring all requirements to be installed). 

https://pip.pypa.io/en/stable/user_guide/#constraints-files